doc/book/getting-started/routing-and-controllers.md

```diff
@@ -49,9 +50,9 @@ return [
     'router' => [
         'routes' => [
             'album' => [
-                'type'    => 'segment',
+                'type'    => Segment::class,
                 'options' => [
-                    'route'    => '/album[/:action[/:id]]',
+                    'route' => '/album[/:action[/:id]]',
```

Comment:
Why it was good? It was in the same column as `defaults` but not the same as `constraints`... so maybe even these 3 values should be aligned? It is very inconsistent at all, we can always align key-value pairs, but usually it looks odd for big, multidimensional arrays.